src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -16,6 +15,9 @@
 using Azure.Core;
 using Operation = AutoRest.CSharp.Input.Operation;
 using static AutoRest.CSharp.Output.Models.MethodSignatureModifiers;
+using System.Runtime.CompilerServices;
```

Comment:
For coding style, do we need to sort the 'using'?

Reply:
Fixed